### ISSUE TITLE
Add a new organization to ROR: 
 
Phanes Biotech 

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#504]
Name of organization: Phanes Biotech
Website: www.phanesbiotech.com
Link to publications: 10.1002/ana.26252
Organization type: Company (A private for-profit corporate entity involved in conducting or sponsoring research)
Wikipedia page: 
Wikidata ID: 
ISNI ID: 
GRID ID: 
Crossref Funder ID: 
Aliases: PBio
Labels:
Acronym/abbreviation:
Related organizations: 
City: Philadelphia 
Country: United States of America
Geonames ID: 4560349
Year established: 2020
How will a ROR ID for this organization be used? To identify affiliations for research publications, To acknowledge research funders

Other information about this request:

**Comment by adambuttrick** [comment]
Biotechnology company whose primary focus is a novel Alzheimer therapeutic. Principals are all well-credentialed with solid publication history (https://phanesbiotech.com/publications/), but do not appear to use Phanes Biotech as their primary affiliation. That being said, is clearly involved with drug development and research and we have many similar companies in ROR (e.g. https://ror.org/02eb03q95 ; Neovii Biotech (Germany)).

**Comment by arthurpsmith** [comment]
Hmm, I think we should wait on this one. There's a very similarly named company that appears to be larger and more well-established, "Phanes Therapeutics" - https://www.phanesthera.com/ (from San Diego, established 2016) - for which we do not have a ROR ID, and it seems likely if we create one for this company without creating one for the other that could lead to some confusion. I can't find any relationship between the two, and this company is not listed for example in Crunchbase or mentioned in any news reports I could find.

The publications listed on their "publications" page are all from 2017 or earlier as far as I could see, which predates the company formation (so it wouldn't have been used as an affiliation). I recommend we put this on hold until we see they are actually publishing or funding research with this affiliation.

**Comment by adambuttrick** [comment]
Moving to hold for later to see if further evidence of affiliation usage surfaces.

**Comment by adambuttrick** [comment]
Still no examples of affiliation usage, so declining as out of scope. May resubmit when affiliation usage can be demonstrated.
